app/views/home/index.html.erb

```diff
@@ -1 +1,17 @@
-<h>Hello World</h>
+<main class="home">
+  <header class="home__header"><a href="#" class="home__logo">ground rules</a></header>
+  <%= render 'svg/home-ill-cup' %>
+  <section class="home__section">
+      <h1>Created for tired people, by slightly less tired people.</h1>
```

Comment:
This comment only applies to this page -- you've got the right idea for the other `<h1>`s in the site